src/ios/Calendar.m

```diff
@@ -392,8 +392,12 @@ - (NSMutableArray*) eventsToDataArray: (NSArray*)matchingEvents {
     if (event.location != nil) {
       [entry setObject:event.location forKey:@"location"];
     }
-    if (event.notes != nil) {
-      [entry setObject:event.notes forKey:@"message"];
+    if (event.notes != nil || event.URL != nil) {
+      NSString *notes = event.notes ?: @"";
+      NSString *url = (NSString*)event.URL ?: @"";
```

Comment:
NSURL can't be cast to NSString.

https://stackoverflow.com/questions/8082719/convert-an-nsurl-to-an-nsstring

Reply:
Its working, but will use event.URL.absoluteString